src/state/apis/snapshot/snapshot.ts

```diff
@@ -224,23 +226,25 @@ export const snapshotApi = createApi({
             }, new Set()),
           )
           const delegation = false // don't let people delegate for discounts - ambiguous in spec
-          const votingPowerResults = await Promise.allSettled(
-            evmAddresses.map(async address => {
-              const votingPowerUnvalidated = await getVotingPower(
-                address,
-                '1',
-                strategies,
-                THOR_TIP_014_BLOCK_NUMBER,
-                THORSWAP_SNAPSHOT_SPACE,
-                delegation,
-              )
-              // vp is THOR in crypto balance
-              return bnOrZero(VotingPowerSchema.parse(votingPowerUnvalidated).vp)
-            }),
+
+          const votingPowerResults = await getVotingPower(
+            evmAddresses,
+            '1',
+            strategies,
+            THOR_TIP_014_BLOCK_NUMBER,
+            THORSWAP_SNAPSHOT_SPACE,
+            delegation,
           )
-          const thorHeld = BigNumber.sum(
-            ...votingPowerResults.filter(isFulfilled).map(r => r.value),
-          ).toNumber()
+
+          const scores = ScoresSchema.parse(votingPowerResults).scores
```

Comment:
Suggestion: Lift this 2d reduction into `getVotingPower` for dryness

Reply:
Cool, fixed in [0275bec](https://github.com/shapeshift/web/pull/8057/commits/0275becb4da42956c26e33f854a40e300206058a)